### PR TITLE
First version

### PR DESCRIPTION
**Pull request by klausenbusk**
README rendered: https://github.com/distributed-technologies/filemover-poc/tree/initial-version#readme

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+
+		signedFiles := make(map[string][]string, 0)
+		for _, file := range files {
+			if file.Mode().IsRegular() && strings.HasSuffix(file.Name(), ".sig") {
+				log.Print("Found signature file: " + file.Name())
+				signedFile := strings.TrimSuffix(file.Name(), ".sig")
+				index := strings.LastIndex(signedFile, ".")
+				if index > 0 {
+					signedFile = signedFile[:index]
+				}
```
Comment (skovbakke) [inline main.go:142]: Not sure I understand this. Do we insist the signatures to have a certain format? and what is this format? If we have a file `foo.bar` it seems that the signature file must be something like `foo.bar.<initials>.sig`.

**Review by skovbakke — CHANGES_REQUESTED** [review]
Overall it looks good. Just a few remarks 🤠

**Comment by skovbakke** [comment]
Also, it seems you forgot to follow the ["setup of repository"-guide](https://github.com/distributed-technologies/wiki/blob/main/git/repo_creation.md) from the wiki. Please, rename the head branch to main and add the branch protection rules. 😊

File: main.go (hunk 1 above)
Comment (klausenbusk) [reply main.go:142]: `foo.bar.<initials>.sig` is correct.

File: main.go [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+
+			for _, sig := range value {
+				sig = fmt.Sprintf("%s/%s", input, sig)
+				entity, err := checkSignature(key, sig)
+				if err != nil {
+					log.Print("Error checking signature " + err.Error())
+					continue
+				}
+				keyId := entity.PrimaryKey.KeyIdString()
+				group := keys[keyId].Group
```
[the remark this answers is not in the document]
Comment (klausenbusk) [reply main.go:167]: The `checkSignature` function will fail.

> CheckDetachedSignature takes a signed file and a detached signature and returns the signer if the signature is valid. If the signer isn't known, ErrUnknownIssuer is returned.

https://pkg.go.dev/golang.org/x/crypto/openpgp#CheckDetachedSignature

**Review by skovbakke — APPROVED** [review]
LGTM 🎉

**Review by jacobcrawford — APPROVED** [review]
Since the functionality has been approved by the owner, and it is a POC, I don't think it makes much sense to make improvements. The current state is acceptable.

**Pull request merged**